examples/explore_browser_use/planner.py

```
@@ -0,0 +1,57 @@
+from browser_use.llm import ChatOpenAI
+from browser_use import Agent, BrowserSession
+from dotenv import load_dotenv
+from PIL import Image, ImageFont
```

Ruff (0.11.9)

`PIL.Image` imported but unused

Remove unused import

(F401)